portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,18 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Adds a random fact about myself to the page.
```

Comment:
Consider comment this function like this: https://google.github.io/styleguide/jsguide.html#jsdoc-method-and-function-comments